samples/snippets/revoke_dataset_access_test.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .. import update_dataset_access
```

Comment:
As mentioned in the comment, adding `samples/snippets/__init__.py` should fix this, but it'll break the other sample tests. You'll need to also update the tests in `samples/snippets/` to use relative imports.

Alternatively, we can start making progress on migrating the snippets to `samples/snippets` (https://github.com/googleapis/python-bigquery/issues/790). You could make a copy of `update_dataset_access.py` and it's corresponding tests and put them in the `samples/snippets` directory in this PR. Then once you've pointed cloud.google.com at the new sample location, send a PR to remove the duplicate sample.

Reply:
I like the latter approach!

Reply:
done in latest commit!